lib/stacker/stack.rb

```diff
@@ -185,6 +185,23 @@ def describe_change_set
           sleep ((6 - retries) * 3)
         end
       end
+      resp_status = ""
+      while resp_status != "CREATE_COMPLETE"
+        resp = region.client.describe_change_set(
+          change_set_name: change_set,
+          stack_name: name,
+          
+        )
+        resp_status = resp.status
+        if ["DELETE_PENDING", "DELETE_IN_PROGRESS", "DELETE_COMPLETE", "DELETE_FAILED", "FAILED"].include? resp_status
+          raise Error.new resp.status_reason
```

Comment:
let's use the the `CannotDescribeChangeSet` error here as the loop above does (`raise CannotDescribeChangeSet.new ...`)